### DISCUSSION TITLE
Whats the agreed way of doing git stuff?

### DISCUSSION THREAD

**Discussion by shelldandy** [LazyVim/LazyVim#2328] [Q&A] [answered]
Seems like fugitive is not installed, there probably is something already so im wondering whats the workflow to add/commit/push

**Comment by cerebrl** [comment, marked as answer]
I think the recommended Git client from LazyVim's perspective is LazyGit; it's listed on the front page of LazyVim's website. It's what I use, and I have no complaints. This is coming from someone that has used the Git CLI exclusively for a over a decade and hated all Git clients. 

I've heard people love Fugitive, so if that's what works for you, install it and don't look back :)

**Reply by shelldandy** [reply]
Thank you now I also use and love lazygit
